CMakeLists.txt

```diff
@@ -119,6 +122,21 @@ cmake_dependent_option(STATIC_LIBS "Build static libraries instead of shared-obj
 option(VERBOSE_LOGS "Adds verbose loging to DALI" OFF)
 option(WERROR "Treat all warnings as errors" OFF)
 
+cmake_dependent_option(DALI_CLANG_ONLY "Compile DALI using only Clang. Suitable only for developement."
+    OFF "CMAKE_CXX_COMPILER_ID STREQUAL Clang" OFF)
+
+if (DALI_CLANG_ONLY AND BUILD_NVDEC)
+  message(STATUS "NVDEC is not supportet when compiling only with Clang. Setting BUILD_NVDEC to OFF.")
```

Comment:
Anything special in NVDEC that doesn't work?

Reply:
Yes, the texture sampling is not implemented in clang.

Reply:
OK